build/storage/tests/python_system_tools/config.py

```diff
@@ -0,0 +1,43 @@
+from abc import ABC, abstractmethod
+
+class BaseConfig(ABC):
+
+    @abstractmethod
+    def __init__(self):
+        pass
+
+
+class StorageTargetConfig(BaseConfig):
+
+    def __init__(self):
+        self.username = ""
+        self.password = ""
+        self.ip_address = ""
+        self.port = ""
+
+
+class IPUStorageConfig(BaseConfig):
+
+    def __init__(self):
+        self.username = ""
+        self.password = ""
+        self.ip_address = ""
+        self.port = ""
+
+
+class SenderConfig(BaseConfig):
```

Comment:
As I Understand `SenderConfig` will represent the container where we have our tests running. It will likely be a local one and I don't know if we need this username/passwords etc. I think we can delete it at this moment

Reply:
deleted